core/language/en-GB/Help/listen.tid

```diff
@@ -18,7 +18,7 @@ All parameters are optional with safe defaults, and can be specified in any orde
 * ''anon-username'' - the username for signing edits for anonymous users
 * ''username'' - optional username for basic authentication
 * ''password'' - optional password for basic authentication
-* ''authenticated-user-header'' - optional name of header to be used for trusted authentication
+* ''authenticated-user-header'' - optional name of header field to be used for trusted authentication.
```

Comment:
The docs do not use the term "field" for headers to avoid confusion with tiddler fields. We could say "request header" perhaps?